torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -554,6 +554,18 @@ void Reducer::delay_all_reduce() {
     }
   }
 
+  if (ddp_debug_level_ == c10d::DebugLevel::Info || ddp_debug_level_ == c10d::DebugLevel::Detail) {
+    for (const auto& unused_index: unused_parameters_) {
```

Comment:
nit: should pass primitive `size_t` by value?

Reply:
I also think pass by & may not be necessary, however I decided to be consistent with other usages of unused_index and unused_parameters_ in reducer.cpp file, e.g. https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/c10d/reducer.cpp#L615, https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/c10d/reducer.cpp#L837, 
should I delete these "&" altogether?

Reply:
No worries then, and no need to delete the other usages.